xbmc/interfaces/Builtins.cpp

```diff
@@ -234,6 +234,7 @@ const BUILT_IN commands[] = {
   { "ToggleDebug",                false,  "Enables/disables debug mode" },
   { "StartPVRManager",            false,  "(Re)Starts the PVR manager" },
   { "StopPVRManager",             false,  "Stops the PVR manager" },
```

Comment:
it would be nice if you could create a separate commit and rename these built-ins to `PVR.StartManager` and `PVR.StopManager`. We should group all PVR related built-ins like we do for other built-ins. Thanks.